packages/css/accordion.css

```diff
@@ -1,139 +1,123 @@
-.ds-accordion {
-  --dsc-accordion-border-radius: min(1rem, var(--ds-border-radius-md));
-  --dsc-accordion-border-color: var(--ds-color-neutral-border-subtle);
+.ds-accordion-group {
   --dsc-accordion-background: var(--ds-color-neutral-background-default);
-  --dsc-accordion-button-background: var(--ds-color-neutral-background-default);
-  --dsc-accordion-button-background-open: var(--ds-color-neutral-background-subtle);
-  --dsc-accordion-icon-background-hover: var(--ds-color-neutral-surface-default);
-  --dsc-accordion-icon-background-active: var(--ds-color-neutral-surface-default);
-
-  border-bottom: 1px solid var(--dsc-accordion-border-color);
-  box-sizing: border-box;
-  background-color: var(--dsc-accordion-background);
-}
-
-.ds-accordion--border {
-  border: 1px solid var(--dsc-accordion-border-color);
-  border-radius: var(--dsc-accordion-border-radius);
-}
+  --dsc-accordion-border-color: var(--ds-color-neutral-border-subtle);
+  --dsc-accordion-border-radius: min(1rem, var(--ds-border-radius-md));
+  --dsc-accordion-chevron-background--active: var(--ds-color-neutral-surface-default);
+  --dsc-accordion-chevron-background--hover: var(--ds-color-neutral-surface-default);
+  --dsc-accordion-chevron-gap: var(--ds-spacing-2);
+  --dsc-accordion-chevron-size: var(--ds-spacing-6);
+  --dsc-accordion-heading-background--open: var(--ds-color-neutral-background-subtle);
+  --dsc-accordion-heading-background: var(--ds-color-neutral-background-default);
+  --dsc-accordion-padding: var(--ds-spacing-4);
+
+  &[data-ds-color="subtle"] {
```

Comment:
Probably forgot to remove the `ds` from data attr after decided to go pure data attr? :)